quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -44,11 +52,37 @@
      */
     @ResponseBody
     @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @RequestHeader(name = "Authorization",  required = true) String authHeader) {
```

Comment:
the authorization should happen through spring security.  This won't be the only api endpoint and we don't want to reimplement the same thing over and over again. See here for an idea how to do that:
https://github.com/diging/giles-eco-giles-web/blob/develop/giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/config/SecurityContext.java#L168